<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="go_back">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>107</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_country">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Select single country</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_dataset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Select entire region/dataset</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>701</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="country_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>country...</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataset_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dataset...</string>
   </property>
  </widget>
  <widget class="QLabel" name="country_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Azerbaijan:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset stats:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QLabel" name="male_count">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
  <widget class="QLabel" name="female_count">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_country_csv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Select country from csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_dataset_csv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Select region/dataset from csv</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
